ghost/admin/app/routes/editor/edit.js

```diff
@@ -31,7 +31,8 @@ export default class EditRoute extends AuthenticatedRoute {
 
         let query = {
             // eslint-disable-next-line camelcase
-            id: post_id
+            id: post_id,
+            include: 'tags,authors,authors.roles,email,tiers,newsletter,count.clicks,post_revisions,post_revisions.author'
```

Comment:
This is gross tbh - it would be nice to skip setting the `include` here and just request the revisions when opening the history feature